apps/nowcasting-app/components/charts/delta-view/delta-view-chart.tsx

```diff
@@ -407,23 +401,21 @@ const GspDeltaColumn: FC<{
             <>
               <div className="pb-0.5">
                 <div
-                  className={`bg-ocf-delta-1000 transition duration-200 ease-out hover:bg-ocf-gray-700 hover:ease-in`}
+                  className={`bg-ocf-delta-950 transition duration-200 ease-out hover:bg-ocf-gray-700 hover:ease-in`}
```

Comment:
I think this colour still should have a home somewhere else - is it delta-specific? It's not part of the "heatmap" sort of colour range, it's just a new grey - maybe need to shift around our `ocf-gray` array to include this at the darker end perhaps..? (And updated any usages too...)
And I think only `[colour-name]-50` is usually the only non-100x multiple in Tailwind convention, which I think is neat and worth sticking to unless absolutely unavoidable 👍